distribution/openhabhome/start.sh

```diff
@@ -14,6 +14,7 @@ cp=$(find $eclipsehome -name "org.eclipse.equinox.launcher_*.jar" | sort | tail
 
 echo Launching the openHAB runtime...
 java \
+	-Dorg.osgi.framework.bundle.parent=ext \
```

Comment:
hmm … changing the ClassLoader behaviour of the whole openHAB instance feels like a fundamental change. Any chance to make this change more focussed (see https://www.eclipse.org/forums/index.php/t/217277/)?